File: Binary Search I/cpp/Day 3/DistanceValue.cpp
class Solution {
public:
    int findTheDistanceValue(vector<int>& arr1, vector<int>& arr2, int d) {
        /*
        Still feguring out how how to apply binary search 🤔
        */
      int count=0;
      for(int i=0;i<arr1.size();i++){
          for(int j=0;j<arr2.size();j++){
              int v=arr1[i]-arr2[j];
              if(v<0){
               v=-v;
              }
              if(v<=d){
                  break;
              }
              if(j==arr2.size()-1){
                  count++;
              }
          }
      }
      return count;
    }
};

File: Binary Search I/python/Day 3/distance_value.py
class Solution:
    def findTheDistanceValue(self, arr1: List[int], arr2: List[int], d: int) -> int:
      '''
      Still feguring out how how to apply binary search 🤔
      '''
      count=0
      for i in range(len(arr1)):
          for j in range(len(arr2)):
              v=arr1[i]-arr2[j]
              if(v<0):
                v=-v
              if(v<=d):
                  break
              if(j==len(arr2)-1):
                  count+=1
      return count